### DISCUSSION TITLE
How do you use this to install Mac OS

### DISCUSSION THREAD

**Discussion by YonathanZzZ** [kalkmann/Legion-5600H-Hackintosh#33] [Q&A]
Hi kaklmann and thank you for making this project public. I have a Legion 5 laptop with a 5800H and RTX 3050 Ti.
I've modified the config.plist to match my CPU's number of cores and also removed Radeon specific kexts and added the NootedRed kext that should enable the AMD iGPU.

When I boot into the USB drive, I see the GUI for selecting a drive to boot. I only see my Windows drive. After pressing space, I also see an option called Rest NVRAM (which I clicked out of curiosity). Should there be a third option to boot into the installer itself? I do have a com.apple.recovery.boot folder in the USB drive that contains two files (BaseSystem.dmg and BaseSystem.chunklist).

I'd appreciate your help. Thank you.

**Comment by artickc** [comment]
```
<key>ScanPolicy</key>
<integer>0</integer>
```
In EFI/OC/config.plist
